docs/.gitignore

```diff
@@ -27,5 +27,3 @@ yarn-error.log*
 /docs/developers/reference/smart_contract_reference/aztec-nr
 /docs/docs/protocol-specs/public-vm/gen
 test-results
-
-versions.json
```

Comment:
versioning is no longer managed by the CI: we're supposed to change this file to control which versions are built and shown on the website